pygpu/gpuarray.pyx

```diff
@@ -510,6 +510,8 @@ def init(dev, sched='default', disable_alloc_cache=False):
     :type sched: {'default', 'single', 'multi'}
     :param disable_alloc_cache: disable allocation cache (if any)
     :type disable_alloc_cache: bool
+    :param single_stream: enable single stream mode
```

Comment:
I didn't understood at first the docstring for sched. Can you tell a little bit more about it? From the flag it cause later, I think it is for single vs multi threads. You could link to nvidia doc if it help.